builds/jazz-build-module/utility-loader.groovy

```diff
@@ -171,4 +171,30 @@ def isReplayedBuild() {
   currentBuild.rawBuild.getCauses().any{ cause -> cause.toString().contains(replayClassName) }
 }
 
+/*
+* Getting the required account
+*/
+def getRequiredData(service_config){
```

Comment:
give a more contextual name to the function

Reply:
changes the name to getAccountInfo